lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -222,6 +222,43 @@ def test_student_view_data_param(self):
                 block_data['type'] in self.BLOCK_TYPES_WITH_STUDENT_VIEW_DATA
             )
 
+    def test_extra_field_when_requested(self):
```

Comment:
It would be helpful if this test had a docstring, and mentioned that the `test` django settings include `other_course_settings ` and `course_visibility ` in `COURSE_BLOCKS_API_EXTRA_FIELDS`.

Without that, I was confused as to how `course_visibility` was working.

Same for the next test.